libs/blocks/marquee-anchors/marquee-anchors.css

```diff
@@ -0,0 +1,179 @@
+html {
+	scroll-behavior: smooth;
+}
+
+:target::before {
```

Comment:
If this adjusts for the gnav then it seems it would be better to add this there and fix the root cause.

Reply:
This is to give the target anchor element a padding offset above its scroll position so the content isn't right at the top edge. It's not critical, lmk if you think it's overstepping or any concerns w/ other block conflicts.  This selector could also benefit accessibility by styling the targeted headline.

Reply:
I like that it pushes it down a bit.  One very minor thing is that anchor links will behave differently when an anchor-marquee is on the page vs when that css hasn't loaded.  However anchor links in Franklin/Milo are pretty broken anyways so for now I don't think it's a concern

Reply:
We should maybe use this this property `scroll-margin-top` on the header elements that have the id on them. Super simple.

https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-margin-top

Reply:
>This is to give the target anchor element a padding offset above its scroll position so the content isn't right at the top edge. It's not critical, lmk if you think it's overstepping or any concerns w/ other block conflicts. This selector could also benefit accessibility by styling the targeted headline.

I think we definitely need something like this, otherwise the header/title element will go behind the gnav/breadcrumbs.

I think maybe applying it to the `<h>` element, like I mentioned above, might be a better solution.